src/central_limit_theorem/SampleForm.jsx

```diff
@@ -57,7 +57,7 @@ export const SampleForm = ({
                     <div className="form-row">
                         <label htmlFor="numberOfSamples"
                             className="float-right">
-                            Number of Samples:</label>
+                            Select a Number of Samples:</label>
```

Comment:
"the" number of samples?

Reply:
@sdreher all good points, the newest commit will fix these.